src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
 'use strict';
 
-// write code here
+const pageBody = document.querySelector('body');
+const tableHeader = document.querySelector('thead');
+const tableHeaderCells = tableHeader.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+let isSecondClick = false;
+const toNumber = (string) => {
+  if (typeof string === 'string' && string.length !== 0) {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum, 10);
+  }
+};
+let clickedSortTable;
+
+tableHeaderCells.addEventListener('click', (e) => {
+  const headerChildren = [...tableHeaderCells.children];
+  const columnName = e.target.closest('th');
```

Comment:
You should check if 'e.target' is actually a 'th' element before calling 'closest', to avoid unnecessary function calls.